### PR TITLE
Added better README and a potential proxy rule

### PR DESCRIPTION
**Pull request by PanopticaRising**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,18 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Rederly Frontend
 
-## Available Scripts
+![Version](https://img.shields.io/github/v/release/rederly/frontend?style=plastic)
```
Comment (red-tt-bear) [inline README.md:3]: How is this getting it's status, is this hooked up to the github action?

File: src/setupProxy.js [hunk 2]
```diff
@@ -10,4 +10,8 @@ module.exports =  function(app) {
     app.use(createProxyMiddleware('/webwork2_files', {
         target: 'http://localhost:3000'
     }));
+
+    app.use(createProxyMiddleware('/work', {
```
Comment (red-tt-bear) [inline src/setupProxy.js:14]: I think we should add dev configuration, possibly as environment variables if it doesn't affect the build

Not now but in the future we should think through howe we test the lambda in development

I think @Jhuanderson-Rederly said there is a way to run it locally, hopefully in docker

File: README.md (hunk 1 above)
Comment (PanopticaRising) [reply README.md:3]: This is getting it from our Releases tab.

File: src/setupProxy.js (hunk 2 above)
Comment (PanopticaRising) [reply src/setupProxy.js]: I just finished testing it and it didn't seem to work well with the redirect to stage, so I just removed it.

**Review by red-tt-bear — APPROVED** [review]

**Pull request merged**